libfreerdp/core/update.c

```diff
@@ -2139,6 +2139,17 @@ void update_free(rdpUpdate* update)
 		free(update->primary);
 		free(update->secondary);
 		free(update->altsec);
+		free(update->window->monitored_desktop.windowIds);
+		free(update->window->window_state.titleInfo.string);
```

Comment:
OK, I will create a function to free window_state and call it at multiple place.